ffmpeg.h

```diff
@@ -25,6 +25,12 @@
 
 #endif
 
+/*
+ * AVPixelFormat is a newer release of avutil PixelFormat is the old name
+ */
+#ifndef AVPixelFormat
```

Comment:
I don't think this will work; at least in the AVUtil I have (debian stable) AVPixelFormat is an enum, not a #define.

My guess would be that #if LIBAVUTIL_VERSION_MAJOR < some-number is the right way, but I don't know that the number is.